addons/link_tracker/models/link_tracker.py

```diff
@@ -168,6 +168,8 @@ def create(self, vals_list):
             if 'url' not in vals:
                 raise ValueError(_('Creating a Link Tracker without URL is not possible'))
 
+            if vals['url'].startswith(('?', '#')):
```

Comment:
Why not using an "api.constrains" ? Seems to do the job ?

Reply:
we cannot do that, the call to `tools.validate_url` will change it is `http://?` which is parsed by urlilb as follow: `SplitResult(scheme='http', netloc='', path='', query='', fragment='')`. We have to validate the URL before we call `tools.validate_url`

Reply:
Oh indeed, there is another processing afterwards. Seems legit then.

Note that you could add a constraint quand même but that would be redundant.